src/index.jsx

```diff
@@ -103,27 +103,38 @@ function bootstrap() {
 function render(Component) {
   const root = document.getElementById('react');
   const { configuration } = store.getState();
-  const googleAuthority = 'https://accounts.google.com';
-  // TODO once we no longer query BQ directly, we'll no longer need the BQ scope
+  const isGoogleAuthority = configuration.configObject.authorityEndpoint.startsWith('https://accounts.google.com');
+  const metadata = {
+    authorization_endpoint: `${window.origin}/oauth2/authorize`,
+    token_endpoint: `${window.origin}/oauth2/token`
+  };
+
   const scopes = ['openid', 'email', 'profile'];
-  if (configuration.configObject.authorityEndpoint === googleAuthority) {
+  console.log(isGoogleAuthority, configuration.configObject.authorityEndpoint);
+  if (isGoogleAuthority) {
+    // TODO once we no longer query BQ directly, we'll no longer need the BQ scope
     scopes.push('https://www.googleapis.com/auth/bigquery.readonly');
     scopes.push('https://www.googleapis.com/auth/drive.file');
+
+    metadata['userinfo_endpoint'] = 'https://openidconnect.googleapis.com/v1/userinfo';
   }
   const oidcConfig = {
     authority: configuration.configObject.authorityEndpoint,
     client_id: configuration.configObject.oidcClientId,
     // overwrite the auth endpoint to use the one hosted by TDR
-    metadata: {
-      authorization_endpoint: `${window.origin}/oauth2/authorize`,
-      token_endpoint: `${window.origin}/oauth2/token`,
-    },
+    metadata,
     redirect_uri: `${window.origin}/redirect-from-oauth`,
     prompt: 'login',
     scope: scopes.join(' '),
+    loadUserInfo: isGoogleAuthority,
     userStore: new WebStorageStateStore({ store: window.localStorage }),
+    extraQueryParams: {'access_type': 'offline'},
   };
 
+  // Log oauth client events
+  Log.setLevel(Log.WARN);
+  Log.setLogger(console);
```

Comment:
Please make sure we don't log the tokens accidentally, even if it's just via `console.log` :)

Reply:
Sanity check - Anything leftover from when this was a draft?

Reply:
Was this just for debugging?